docs/start/requirements.md

```diff
@@ -19,15 +19,44 @@ huronOS works as a live USB, so, in your requirements, you will need to consider
+
+- **Disable IPv6**  
+    huronOS is currently not capable of supporting IPv6 automated firewall setup. Due to this limitation, it is recommended to disable IPv6 on your network and use only IPv4. Having a dual IP stack for v4 and v6 can lead to IPv4 firewall not working properly, as users might have access to IPv6-allowed sites.
+
+- **Wired Network**  
+    While WiFi is supported, a wired network is always preferred over a wireless network for a more reliable and automated setup. Additionally, wireless cards come from a variety of vendors, and the drivers might not be available with the huronOS ISO image. We strive to provide as many drivers as possible, but sometimes you will need to install them manually. With Ethernet, this isn't the case as most generic drivers work on all types of Ethernet interfaces.
+
+- **Firewall**:  
+    - **ICMP** *[all ICMP traffic]*  
+        ICMP protocol is unoffensive but very useful for performing pings and is a good indicator that the network is working and properly configured.
+
+    - **DNS** *[Source port 53: UDP, TCP traffic]*  
+        DNS is essential for web browsing and for setting up huronOS's firewall.
+        > **Note:** huronOS uses Google's DNS servers (*8.8.8.8* and *8.8.4.4*) as the main DNS service and Cloudflare's DNS servers (*1.1.1.1* and *1.0.0.1*) as a backup service. This can be overridden by setting a *nameserver* when using DHCP. Please be aware of network slowness if your network-level firewall is blocking either Google DNS or Cloudflare DNS.
+
+    - **DHCP** *[Source port 67, Dest port 68: UDP]*  
+        DHCP is necessary to avoid manually configuring the network on huronOS. It is recommended to keep these ports open even if manual setup is being performed. However, if you want to quickly set up an arena with huronOS, your setup time can be drastically reduced with DHCP.
+
+    - **NTP** *[Source port 123: UDP]*  
+        NTP is **extremely important** for huronOS, as several features rely on accurate time and date, such as starting or finishing a contest mode. The only way to ensure that all computers are in sync and none of them will start a contest at a different time is by using NTP to synchronize the date.
+        > *By default, huronOS does not trust the hardware clock, so having the NTP service is essential.*
+
+    - (Optional) **Translate API** *[Source port: 443, Source web: translate.googleapis.com]*  
+        If you will be running a contest for participants with a very early proficiency in English (which is the official language for several competitions), you might want to provide a translator (that cannot act as a proxy) to help them better understand the problems. This firewall rule is necessary for *Crow Translate* to work. Make sure to [enable this software](../usage/directives/configurations/software-modules.md) in the directives.
```

Comment:
I think the correct way to express this idea would be "very basic proficiency" and even better would be "elementary proficiency".